src/handleActions.js

```diff
@@ -3,16 +3,18 @@ import reduceReducers from 'reduce-reducers';
 import invariant from 'invariant';
 import handleAction from './handleAction';
 import ownKeys from './ownKeys';
+import { flattenReducerMap } from './namespaceActions';
 
 export default function handleActions(handlers, defaultState) {
```

Comment:
I'm guessing most people wouldn't even want to use the custom namespace property anyway. We can add later if requested.